src/lib/Session.js

```diff
@@ -1,9 +1,9 @@
 import { withIronSessionApiRoute, withIronSessionSsr } from 'iron-session/next';
-import { log } from '../lib/Logger';
-import { slsFetch } from "../lib/Client";
+import { log } from './Logger';
+import { slsFetch } from "./Client";
```

Comment:
I preferred the one that explicitly mentioned that the library belonged to `lib`, rather than the developer having to check which folder `Session` is.

```suggestion
import { log } from '../lib/Logger';
import { slsFetch } from "../lib/Client";
```